CONTRIBUTING.md

```diff
@@ -56,7 +56,9 @@ As you increase your knowledge of OpenRewrite, you might find mistakes in our do
 
 The best way to work towards a fix is to first replicate the issue with a minimal unit test. Nearly every recipe has a dedicated unit test that shows a before and after situation. Usually, you can copy the first unit test for any specific recipe, and adjust that to match the problem case. 
 
-Once replicated, please open a draft pull request, even if you are unsure whether or not you will be able to contribute a fix. That allows us to also have a look at the specifics of the issue in our IDE, if necessary. From there, we generally recommend that people step through the recipe execution using the debugger while running the new unit test. As you make changes to the recipe, you should see the test go from failing to passing. If you get stuck at any point, we’re happy to help.
+Once replicated, you can already open a draft pull request, even if you are unsure whether you will be able to contribute a fix. That allows us to also have a look at the specifics of the issue in our IDE, if necessary. From there, we generally recommend that people step through the recipe execution using the debugger while running the new unit test. As you make changes to the recipe, you should see the test go from failing to passing. If you get stuck at any point, we’re happy to help.
```

Comment:
Was the change `you can already open` due to some confusion with the existing text? I feel like `please open a draft pull request` is more direct and clear on what action someone needs to take but maybe I'm missing something?

Reply:
Yes that was in response to [this question](https://rewriteoss.slack.com/archives/C01A843MWG5/p1685033627861109), and the thread that follows it.. Want I wanted to distinguish is that we invite people to open draft pull requests early, to get feedback, yet they can still use the same pull request to contribute the fix as well.

With the previous wording we don't explain why we want people to open pull requests early, and if they can then keep pushing changes.

If based on that you have some other suggestion have at it! I trust your judgement to either roll this back or improve upon it.

Reply:
Interesting -- I'd propose:

```suggestion
Once replicated, please open a draft pull request even if you are unsure whether you will be able to contribute a fix. If you do decide that you want to fix it, you can iterate on that PR. 

By opening one early, you allow us to have a look at the specifics of the issue in our IDE, if necessary. 

From there, we generally recommend that people step through the recipe execution using the debugger while running the new unit test. As you make changes to the recipe, you should see the test go from failing to passing. If you get stuck at any point, we’re happy to help. Please feel free to update the PR with questions.
```